recipe/meta.yaml

```diff
@@ -1,33 +1,40 @@
-{% set version = "0.7.3" %}
+{% set name = "vispy" %}
+{% set version = "0.12.1" %}
 
 package:
   name: vispy
   version: {{ version }}
 
 source:
   fn: vispy-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/v/vispy/vispy-{{ version }}.tar.gz
-  sha256: 1edd3434247577e0910f9c76331b82a54be7e5381c022120c7336e73167b155c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e00881c1d0f9b2c08eb6e26e9361adbde8a3a96e7878ee6c1e199e7c5872224f
   patches:
     - ctypes_fontconfig.diff
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  ignore_run_exports:
+    - fontconfig
+    - python
 
 requirements:
   build:
+    - {{ compiler('c') }}
     - patch     # [not win]
     - m2-patch  # [win]
   host:
     - pip
     - python
-    - setuptools
+    - setuptools >=30.3.0
```

Comment:
setuptools and cython should be without pinnings, https://github.com/vispy/vispy/blob/7accf231d251271eae6edfa3e1b103c15bc0a95e/setup.py#L97

Reply:
I got the requirement from here: https://github.com/vispy/vispy/blob/7accf231d251271eae6edfa3e1b103c15bc0a95e/pyproject.toml#L4
and here:
https://github.com/vispy/vispy/blob/7accf231d251271eae6edfa3e1b103c15bc0a95e/pyproject.toml#L10